WWDC22rawJSON/markdown_output/accessibility-lounge.md

```diff
@@ -1,5 +1,6 @@
 # accessibility-lounge QAs
-#### by [FeeTiki](https://twitter.com/FeeTiki)
```

Comment:
Did you mean to delete yourself in a bunch of these? 😅

Reply:
Agreed, need to add yourself in there man 😅

Reply:
We should just edit the contributors.json and with next run it will autoupdate?